sphinx/writers/texinfo.py

```diff
@@ -761,10 +761,12 @@ def visit_title_reference(self, node: Element) -> None:
     # -- Blocks
 
     def visit_paragraph(self, node: Element) -> None:
-        self.body.append('\n')
+        if not self.in_footnote:
+            self.body.append('\n')
```

Comment:
Is this removal required? It seems Texinfo does not mind if any whitespace exists.

Reply:
Well, the generated texinfo output is more readable and easier for writing a test-case. Excluding the hunk, one would see:

```
@item Footnote in term @footnote{
Footnote in term}
```

Reply:
Finally, I determined to remove this code to keep our writer simple. I think the simpleness of the writer is more important than the readability of output.